src/dev/i18n/extract_default_translations.js

```diff
@@ -114,10 +170,10 @@ export async function extractDefaultTranslations(inputPath) {
   jsonBuffer = Buffer.concat([jsonBuffer, Buffer.from('}\n')]);
 
   try {
-    await accessAsync(resolve(inputPath, 'translations'));
+    await accessAsync(path.resolve(output));
```

Comment:
__nit:__ we can remove the code that creates directories - it's responsibility of the user to provide existing output directory.